README.md

```diff
@@ -27,11 +27,11 @@ Repository to describe, develop, document and test the DeviceStatus API family
 * Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest available public release**.
 
 <!-- following part left in for now but could be omitted going forward -->
-* The latest available and released version 0.5.1 of the API is available within the [release-0.5.1 branch](https://github.com/camaraproject/DeviceStatus/tree/release-0.5.1)
+* The latest available release version 0.6 (with API version 0.6.0-alpha.1) is available within the [release-0.6 branch](https://github.com/camaraproject/DeviceStatus/tree/release-0.6)
```

Comment:
r0.6 contains only a pre-release version of the API version 0.6.0, not the latest public-release initial version (which is still 0.5.1).

You can consider the complete part and replace with the following lines:

```
The latest public release is available here: https://github.com/camaraproject/DeviceStatus/releases/latest
Pre-releases of this sub project are available in https://github.com/camaraproject/DeviceStatus/releases
For changes see [CHANGELOG.md](https://github.com/camaraproject/DeviceStatus/blob/main/CHANGELOG.md)
```